README.md

```diff
@@ -154,6 +154,33 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Pushing Changes to a Remote Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to a remote Cluster for others to review.  This involves building a local image, pushing the image to Quay, then updating a remote cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. Quay Account <br> https://quay.io/signin/
+    - Sign in using GitHub account
+    - Logging into Quay from the Docker CLI
+      - Quay Account Settings - > Generate CLI Password: Generate Encrypted Password
+      ```
+      sudo docker login quay.io
+      Username: userFooBar
+      Password: ( password here)```
+    *Highly recommend going through Quay Tutorial which walks you though logging in with Docker and pushing a test image.
+##### Steps
+1. Create a Quay repository <br> Login to https://quay.io, and create a blank, empty repository.  For this example, we'll use _devchanges_ as our example repo; created under _userFooBar_ quay account.
+2. Build Image <br> `sudo docker build -t devchanges github.com/userFooBar/devchanges`
```

Comment:
`sudo` might not be required depending on your platform. I would probably just leave it something generic like `<your-image>`. Maybe give an example like `quary.io/<your-account>/console:latest`

```suggestion
$ docker build -t <your-image> .
$ docker push <your-image>
```

Reply:
nit: Can you remove the `<br>` throughout? It shouldn't really be necessary in markdown. It's good to use 3 backticks to put example commands in a separate block.

Reply:
ok, 3 backticks introduce a lot of white space/lines, was trying for more condensed format.  I'll make another pass.